mmv1/third_party/terraform/services/storage/data_source_google_storage_bucket_objects.go

```diff
@@ -0,0 +1,158 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+package storage
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+)
+
+func DataSourceGoogleStorageBucketObjects() *schema.Resource {
+	return &schema.Resource{
+		Read: datasourceGoogleStorageBucketObjectsRead,
+		Schema: map[string]*schema.Schema{
+			"bucket": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"delimiter": {
```

Comment:
Looks like this never gets used, should we remove it?

Reply:
@slevenick 
Thanks for the hint!

I'd like to add keep and use it because the change to use it wouldn't be a big deal and the `delimiter` seems to be a good optional parameter to query for bucket objects in regard to the Storage API Reference.

Reply:
@slevenick 
How can I mark the requested change as done?